event_model/__init__.py

```diff
@@ -706,7 +708,9 @@ def pack_event_page(*events):
         time_list.append(event['time'])
         uid_list.append(event['uid'])
         seq_num_list.append(event['seq_num'])
-        filled_list.append(event['filled'])
+        filled_list.append(event.get(
+                           'filled',
+                           {key: False for key in event['data'].keys()}))
```

Comment:
We should only have entries for key that can be filled (that is keys that have an `'external'` field in the descriptor).

Reply:
Updated